gtclang/test/integration-test/CodeGen/CMakeLists.txt

```diff
@@ -77,7 +77,7 @@ function(compile_target)
   endif()
 
   target_compile_definitions(${executable} PRIVATE -DOPTBACKEND=${backend})
-  target_compile_features(${executable} PRIVATE cxx_std_11)
+  target_compile_features(${executable} PRIVATE cxx_std_14)
```

Comment:
@havogt should this be inherited from the `GridTools::gridtools` target?

Reply:
It should be inherited, I'll try to remove it.